src/Server.ts

```diff
@@ -39,6 +44,8 @@ export class Server {
    * @returns {Server}
    */
   public $beforeRoutesInit(): void {
+    Chain.register(ChainNetwork.Ethereum, new Ethereum());
+    Chain.register(ChainNetwork.BinanceSmartChain, new BinanceSmartChain());
```

Comment:
this is a duplicate of the 

```
  public $beforeInit(): void {
    loadChains();
  }
```

above, not sure if there are races here or what was going on for these - temp for now

Reply:
I think we can move these strategies to services later and should clean this up